tests/main/install-store/task.yaml

```diff
@@ -43,5 +43,12 @@ execute: |
     echo "Install a snap that contains bash-completion scripts"
     snap install --edge test-snapd-complexion
 
-    echo "All snap blobs are 0600"
-    test "$( find /var/lib/snapd/{snaps,cache,seed/snaps}/ -type f -printf '%#m\n' | sort -u | xargs )" = "0600"
+    # shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB/systems.sh"
+    # TODO:UC20: fix to work on uc20 too
+    # The "seed/" dir is on a FAT partition on uc20 so the permissions are
+    # different here.
+    if ! is_core20_system; then
```

Comment:
This is not executed on core systems, is it needed?

Reply:
Aha, thank you! I missed that, will remove it again.